docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
```

Comment:
I think the container should have juttle and outrigger already installed and built rather than run `npm` from inside the container.

Then you shouldn't need python or any node-gyp stuff at all.

In other words, the instructions should be to run the install / build on a dev machine that has all the right bits on it, then your dockerfile would take the results of the build (i.e. node_modules) and include them in the image.